src/Subscreen.mc

```diff
@@ -0,0 +1,15 @@
+using Toybox.WatchUi;
+
+module Subscreen {
+
+(:connectiq3,:glance)
+function getSubscreen() {
+    return WatchUi.getSubscreen();
+}
+
+(:connectiq2)
+function getSubscreen() {
```

Comment:
This is unfortunately broken when e.g. building for the vivoactive3 device. I'll try to have a look on how to do the device capability differently.. excluding annotations does not seem to scale well.